protx-client/src/components/ProTx/components/charts/FigureCaption.jsx

```diff
@@ -12,6 +12,6 @@ export const FigureCaption = ({label, captionText}) =>
 
 FigureCaption.propTypes = {
     label: PropTypes.string.isRequired,
-    captionText: PropTypes.string.isRequired,
+    captionText: PropTypes.array.isRequired,
```

Comment:
suggestion: what about keeping it a string.  and use `+` and concat the two parts of the string

Reply:
Keeping it a string was my original intent, however because I needed to pass in html markup (e.g. to make the county name bold) I had to do use this workaround with an array.

Reply:
gotcha.  i understand. 

> One solution would be to use props.children, as you mentioned earlier, however at that point I don't think we gain much by having this component exist at all.

might be good follow on work to consider props.children and then use this for "Table 1" figure caption